neuraxle/base.py

```diff
@@ -1527,6 +1530,39 @@ def __str__(self):
 NamedTupleList = List[Union[Tuple[str, 'BaseStep'], 'BaseStep']]
 
 
+class ForceHandleMixin:
```

Comment:
Suggestion for next time: open a PR with just one such class and with very minimal other accompanying code. 

E.g.: currently, at least 2 other opened PRs depend on this, for a total of 3 PRs depending on that class at the same time.